packages/react-docs/src/pages/components/modal.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Modal, Button } from '@patternfly/react-core';
+import Example from '../../components/example';
+
+class ModalDocs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+      showLargeModal: false
+    };
+
+    this.hideModal = this.hideModal.bind(this);
+    this.showModal = this.showModal.bind(this);
+    this.hideLargeModal = this.hideLargeModal.bind(this);
+    this.showLargeModal = this.showLargeModal.bind(this);
+  }
+
+  showModal() {
+    this.setState({
+      showModal: true
+    });
+  }
+
+  hideModal() {
+    this.setState({
+      showModal: false
+    });
+  }
+
+  showLargeModal() {
+    this.setState({
+      showLargeModal: true
+    });
+  }
+
+  hideLargeModal() {
+    this.setState({
+      showLargeModal: false
+    });
+  }
+
+  render() {
+    const { data } = this.props;
+    return (
+      <ComponentDocs data={data}>
+        <Example title="Simple Modal Box">
+          <Button
+            variant="primary"
+            label="Simple Modal Example"
+            onClick={this.showModal}
+          >
+            Show Modal
+          </Button>
+          <Modal
+            show={this.state.showModal}
+            header="Modal Header"
+            footer="Modal Footer"
+            onClose={this.hideModal}
+            label="Modal Header"
+          >
+            Modal Body Modal Body Modal Body Modal Body Modal Body Modal Body
```

Comment:
I know I sent this previously, but let's use lorem ipsum https://lipsum.com/ or the text from the core examples so we match them.